src/workbooks/governance/workbook.json

```
@@ -6970,99 +7326,620 @@
+                              "formatOptions": {
+                                "palette": "auto"
+                              },
+                              "numberFormat": {
+                                "unit": 17,
+                                "options": {
+                                  "maximumSignificantDigits": 3,
+                                  "maximumFractionDigits": 2
+                                }
+                              }
+                            },
+                            "showBorder": true,
+                            "sortCriteriaField": "type",
+                            "sortOrderField": 1
+                          }
+                        },
+                        "name": "query - PaaS - Automation Detailed"
                       }
                     ]
                   },
-                  "customWidth": "50",
-                  "conditionalVisibilities": [
-                    {
-                      "parameterName": "Subscription",
-                      "comparison": "isNotEqualTo",
-                      "value": ""
-                    },
-                    {
-                      "parameterName": "Location",
-                      "comparison": "isNotEqualTo"
-                    }
-                  ],
-                  "name": "query - ComputeLimits"
+                  "name": "Group - Automation",
+                  "styleSettings": {
+                    "showBorder": true
+                  }
                 },
                 {
-                  "type": 3,
+                  "type": 12,
                   "content": {
-                    "version": "KqlItem/1.0",
-                    "query": "{\"version\":\"ARMEndpoint/1.0\",\"data\":null,\"headers\":[],\"method\":\"GET\",\"path\":\"/subscriptions/{Subscription:id}/providers/microsoft.network/locations/{Location}/usages?api-version=2022-01-01\",\"urlParams\":[],\"batchDisabled\":false,\"transformers\":[{\"type\":\"jsonpath\",\"settings\":{\"tablePath\":\"$.value\",\"columns\":[{\"path\":\"currentValue\",\"columnid\":\"Used\",\"columnType\":\"long\"},{\"path\":\"limit\",\"columnid\":\"Limit\",\"columnType\":\"long\"},{\"path\":\"name.localizedValue\",\"columnid\":\"Resource\"}]}}]}",
-                    "size": 0,
-                    "title": "Network resource limits",
-                    "showRefreshButton": true,
-                    "showExportToExcel": true,
-                    "queryType": 12,
-                    "gridSettings": {
-                      "filter": true,
-                      "sortBy": [
-                        {
-                          "itemKey": "Limit",
-                          "sortOrder": 1
-                        }
-                      ]
-                    },
-                    "sortBy": [
+                    "version": "NotebookGroup/1.0",
+                    "groupType": "editable",
+                    "title": "App Services",
```

Should this be "App Service" as in the service or "App services" as in the resource type?
```suggestion
                    "title": "App services",
```